### PR TITLE
feat: more user_migration stuff

### PR DESCRIPTION
**Pull request by jrconlin**
## Description
* create script to move users by node directly
* moved old scripts to `old` directory (for historic reasons, as well as possible future use)
* cleaned up README to reflect modern reality

## Testing
These are operational scripts. If desired, you can test using the stage: sync_migration_test spanner instance. 

## Issue(s)

Issue: #407

### CODE REVIEW

**Comment by jrconlin** [comment]
using this branch to track changes to the scripts used for migration.

File: tools/user_migration/README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-it's basically "read a row/write a row" only with a middleman system. I wonder if it might
-be better to just iterate through mysql on a thread and write to spanner directly.
-
 There are several candidate scrips that you can use. More than likely, you want to use
 
-`migrate_user.py --dsns <file of database DSNs> --users <file of user ids>
-    [--token_dsn <tokenserver DSN>]`
-
+```bash
+GOOGLE_APPLICATION_CREDENTIALS=credentials.json migrate_node.py \
```
[the remark this answers is not in the document]
Comment (fzzzy) [reply tools/user_migration/README.md]: Thanks for already doing this :)

File: tools/user_migration/README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+    [--dsns=move_dsns.lst] \
+    [--deanon --fxa_file=users.csv] \
```
[the remark this answers is not in the document]
Comment (jrconlin) [reply tools/user_migration/README.md]: yes. I decided not to automatically enable `fxa-file` as a safeguard.
It's the "are you sure" approach to copying user data.

File: tools/user_migration/migrate_node.py [hunk 3]
```diff
@@ -0,0 +1,603 @@
...
+                                values
+                            ))
+                        pass
+        finally:
+            cursor.close()
+
+    def get(self, name, collection_id=None):
+        """Fetches the collection_id"""
+
+        def transact(transaction, values):
```
[the remark this answers is not in the document]
Comment (jrconlin) [reply tools/user_migration/migrate_node.py]: thanks. I left that in before I discovered the "You can't do transactions within transactions" limitation.

**Pull request merged**